onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -2477,4 +2477,18 @@ namespace OperatorHelper
         return outputDimensionsList;
     }
 
+    std::vector<EdgeShapes> ShapeHelper::GetOutputShapes(const MLShapeInferenceContext & shapeInfo) const
+    {
+        std::vector<EdgeShapes> edgeShapes(1);
+        edgeShapes[0] = { shapeInfo.GetInputTensorDimensionCount(0) };
+        return edgeShapes;
+    }
+
+    std::vector<EdgeShapes> SizeHelper::GetOutputShapes(const MLShapeInferenceContext & shapeInfo) const
+    {
+        std::vector<EdgeShapes> edgeShapes(1);
+        edgeShapes[0] = { 1 };
+        return edgeShapes;
+    }
```

Comment:
[minor] I see nearly all the others use this form:

```
return { EdgeShapes(m_outputDimensions) };
```

Does that not work here?

```
return { EdgeShapes({1u}) };
```